404.md

```diff
@@ -4,6 +4,6 @@ permalink: "/404.html"
 layout: page
 ---
 
-![Lukas the bear is lost...]({{ site.baseurl }}/images/404.svg)
+![Lukas the bear is lost...]({{ site.baseurl }}/404.svg)
```

Comment:
His name is Lukas? Didn't know lol

Reply:
This dates back to the old CatSource. Fenks for fix

Reply:
Sorry lol

Reply:
wot